### PR TITLE
feat: fetch variables required to submit the image

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

File: scripts/build-push-all.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@ HELM_CHART_VERSION=v4
 SUMO_REGISTRY="public.ecr.aws/sumologic/"
 PUSH=""
 CHECK="${CHECK:-true}"
+OPERATOR_PROJECT_ID=6075d88c2b962feb86bea730
```
Comment (aboguszewski-sumo) [inline scripts/build-push-all.sh:10]: why do we have a default here and what is it?

Comment (aboguszewski-sumo) [reply scripts/build-push-all.sh:10]: Maybe add a comment here that explains it.

Comment (sumo-drosiek) [reply scripts/build-push-all.sh:10]: It's static :D
I will add `readonly`

Comment (sumo-drosiek) [reply scripts/build-push-all.sh]: done :)

**Review by kkujawa-sumo — APPROVED** [review]
❤️

**Pull request merged**